aip/general/0121.md

```diff
@@ -61,6 +61,10 @@ exposes a large number of resources with a small number of methods on each
 resource. The methods can be either the standard methods ([Get][], [List][],
 [Create][], [Update][], [Delete][]), or [custom methods][].
 
+A resource **must** support at minimum [Get][] and [List][]: clients must be
```

Comment:
This language conflicts with Singleton Resources that must not support List https://google.aip.dev/156 so I'm not sure we can enforce "must" for _List_. I think it has to be the following:
```suggestion
A resource **must** support at minimum [Get][] and **should** support [List][]: clients must be
```

Reply:
I disagree with that approach. If there's a valid exception, we should exempt that case only.

A resource must support Get and all non-singleton resources must support List.

Reply:
Fine, as long as the language doesn't conflict with supporting Singleton resources.

Reply:
added! ptal.